RevenueCatUI/Views/TemplateBackgroundImageView.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  TemplateBackgroundImageView.swift
+//  
+//
+//  Created by Nacho Soto on 8/1/23.
+//
+
+import RevenueCat
+import SwiftUI
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+struct TemplateBackgroundImageView: View {
+
+    var configuration: TemplateViewConfiguration
+
+    var body: some View {
+        if let url = self.configuration.backgroundImageURL {
+            self.image(url)
+                .unredacted()
+                .frame(maxWidth: .infinity, maxHeight: .infinity)
+                .edgesIgnoringSafeArea(.all)
+        }
```

Comment:
Does this need to get wrapped in a `VStack` or something to prevent any of those weird SwiftUI bugs that you've been seeing?

Reply:
Good question, but this one only has one branch and no button to change that state so the but wouldn't apply 😅